src/sentry/monitors/models.py

```diff
@@ -623,8 +623,15 @@ class Meta:
 
     __repr__ = sane_repr("monitor_id", "environment_id")
 
+    def get_environment(self) -> Environment:
+        return Environment.objects.get(id=self.environment_id)
```

Comment:
You might want to use cached reads here as environments don't change that often. The `Environment` model will need to be changed to support caching first though.